applications/GeoMechanicsApplication/custom_conditions/surface_normal_load_3D_diff_order_condition.cpp

```diff
@@ -92,21 +91,16 @@ double SurfaceNormalLoad3DDiffOrderCondition::CalculateIntegrationCoefficient(
     KRATOS_CATCH("")
 }
 
-void SurfaceNormalLoad3DDiffOrderCondition::CalculateAndAddConditionForce(VectorType& rRightHandSideVector,
+void SurfaceNormalLoad3DDiffOrderCondition::CalculateAndAddConditionForce(Vector& rRightHandSideVector,
                                                                           ConditionVariables& rVariables)
 {
-    const SizeType NumUNodes = GetGeometry().PointsNumber();
-    SizeType       Index;
-
-    for (SizeType i = 0; i < NumUNodes; ++i) {
-        Index = i * 3;
-
-        rRightHandSideVector[Index] +=
-            rVariables.Nu[i] * rVariables.ConditionVector[0] * rVariables.IntegrationCoefficient;
-        rRightHandSideVector[Index + 1] +=
-            rVariables.Nu[i] * rVariables.ConditionVector[1] * rVariables.IntegrationCoefficient;
-        rRightHandSideVector[Index + 2] +=
-            rVariables.Nu[i] * rVariables.ConditionVector[2] * rVariables.IntegrationCoefficient;
+    for (SizeType node = 0; node < this->GetGeometry().PointsNumber(); ++node) {
```

Comment:
We don't need `this->` here (or, do we?):
```suggestion
    for (SizeType node = 0; node < GetGeometry().PointsNumber(); ++node) {
```

Reply:
removed.